packages/create-pantheon-decoupled-kit/src/templates/next-drupal-app/package.json.ts

```diff
@@ -8,32 +11,29 @@
 		"start:mono": "next start",
 		"lint": "next lint",
 		"lint:fix": "next lint --fix && npm run prettier:fix",
-		"prettier": "prettier \"**/*.{js,jsx,md}\" --check --ignore-path .prettierignore",
-		"prettier:fix": "prettier \"**/*.{js,jsx,,md}\" --write --ignore-path .prettierignore",
+		"prettier": "prettier '**/*.{js,jsx,md}' --check --ignore-path .prettierignore",
+		"prettier:fix": "prettier '**/*.{js,jsx,,md}' --write --ignore-path .prettierignore",
 		"test": "vitest run",
 		"update-snapshots": "vitest run --update --silent",
 		"coverage": "vitest run --coverage"
 	},
 	"dependencies": {
-		"@pantheon-systems/drupal-kit": "{{drupalKitVersion}}",
-		"@pantheon-systems/nextjs-kit": "{{nextjsKitVersion}}",
-		{{#if tailwindcss}}
-		{{> tailwindcssDeps}}
-		{{/if}}
-        "dotenv": "^16.0.2",
+		"@pantheon-systems/drupal-kit": "${String(data.drupalKitVersion)}",
+		"@pantheon-systems/nextjs-kit":  "${String(data.nextjsKitVersion)}",
```

Comment:
```suggestion
		"@pantheon-systems/drupal-kit": "${data.drupalKitVersion}",
		"@pantheon-systems/nextjs-kit":  "${data.nextjsKitVersion}",
```
These should be typed via the `InputIndex` type. You may need to make the config a template file too in order to get around VSCode picking up that tsconfig. Maybe there's some vscode setting we can use to ignore those nested configs instead but I haven't looked into it.